### ISSUE TITLE
ClickThroughBlocker v0.10.13 and up changed. kOS needs a slight alteration to match.

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#2796]
Starting in ClickThroubBlocker 0.10.13 a new button was added to the toolbar to toggle mode between focus-follows-mouse and focus-follows-click, however this change means kOS no longer is correctly slaving itself to follow CTB's behavior.

I asked a question about this in this issue on CTB, asking if there's a better more 'official' API way to ask for the information from CTB that would be better future-proofed than kOS just reading it from CTB's implementation.

linuxgurugamer/ClickThroughBlocker#19

I may have to wait to fix this until after that question is answered.

In a nutshell the issue is that kOS was reading ClickThroughBlocker's own ``focusFollowsClick`` flag to cause its own behavior to match what CTB's behavior is, but that flag moved when CTB went through a refactor.  I *could* change it in kOS to now try to look in the new place instead, but that could again be broken by the next refactor in CTB.  It seems like it would be more future-proof to have a public-facing place to read it that's an "advertised API" so to speak, rather than having kOS rely on CTB never being refactored again, which isn't a safe assumption.

**Comment by Dunbaratu** [comment]
dangit the PR associated to the wrong issue. re-opening.
